ld-pruning-wf.wdl

```diff
@@ -0,0 +1,440 @@
+version 1.0
+
+# General notes:
+#
+# variables and functions exclusive to the WDL version tend to have "py_" in front
+# and use camelCase between words - this is to make it clear they don't quite 
+# correlate to the CWL
+
+# [1] ld_pruning -- calculates linkage diseq on a GDS file
+task ld_pruning {
+	input {
+		File gds
+		File? sample_include_file
+		File? variant_include_file
+		String genome_build = "hg38"  # can also be hg18 or hg19
```

Comment:
WDL does not have enumerated types, which is why I have to use if/else checks later on down the WDL. I'm down to include parameter_meta{} section but my understanding is that section has no influence on runtime behavior and is just for documentation. I'm also not aware of anything in that section showing up in Terra.

Reply:
On second thoughts... that section is not used in Terra, and all the information I would include is in the documentation already. It also implies things that aren't necessarily true, ie, if I used the suggestions version for the genome_build parameter then a user may incorrectly assume that variable works like an enumeration rather than a string that is checked on the task level (which would have cost implications).

Reply:
https://github.com/DataBiosphere/analysis_pipeline_WDL/pull/15#issuecomment-840912964